input/pagecontent/2-5-EventNotification.md

```diff
@@ -100,9 +81,9 @@ Field | Optionality | Type | Description
 }
 ```
 
-### `webhook` and `websocket` Event Notification Sequence
+### Event Notification Sequence
 
-{% include img.html img="EventNotificationSequence.png" caption="Figure: Event Notification flow diagram" %}
+{%include EventNotificationSequence.svg%}

```

Comment:
Should the sequence diagram also show that the "Driving App" receives the context change event?  I requester gets the event as well as other subscribers, correct?

Reply:
These changes should be made throughout, not just here.